terraform/main.tf

```diff
@@ -126,7 +126,7 @@ locals {
       num_builders_aarch64    = 0
       # 'priv' and 'dev' environments use the same binary cache signing key
       binary_cache_public_key = "ghaf-infra-dev:EdgcUJsErufZitluMOYmoJDMQE+HFyveI/D270Cr84I="
-      binary_cache_url        = "https://ghaf-binary-cache-${local.ws}.${azurerm_resource_group.infra.location}.cloudapp.azure.com"
+      binary_cache_url        = "https://ghaf-binary-cache-${local.ws}.${azurerm_resource_group.infra.location}.cloudapp.azure.com, https://dev-cache.vedenemo.dev"
```

Comment:
This is a bit of a hack and not an URL - this only works "by accident" as caddy accepts this string with commas and whitespaces. Let's add a TODO to clean this up later.

Reply:
I assume long term we only want to have one canonical domain, so this should just highlight the smell. If we start wanting to have multiple domains for an extended period of time, we'd need to make this a list, and do the templating to a comma-and-whitespace-separated list of urls shortly before we pass it to caddy.

Reply:
Thanks, fixed in https://github.com/tiiuae/ghaf-infra/pull/216/commits/040ced3010f4175d244b4f07a74f5ad81ff5837e.